src/index.ts

```diff
@@ -137,4 +137,7 @@ async function main() {
 	}
 }
 
-main().catch(process.exit);
+main().catch((e) => {
+	console.log(e);
```

Comment:
Could we use `console.error`? it pipes to `stderr` instead of `stdout`

Reply:
yes will do. good one